### DISCUSSION TITLE
Renovate for Python pip-requirements should automatically act on common requirements file patterns like requirements-dev.txt

### DISCUSSION THREAD

**Discussion by deniseunt** [renovatebot/renovate#22717] [Suggest an Idea]
### Tell us more.

For Python, only `requirements.txt` (or `.pip`) gets picked up by default, but it's a common pattern to have other requirements files, eg `requirements-dev.txt`

I know that users can configure a fileMatch in pip-requirements per repository.

I'm curious if it was a conscious decision to only match on requirements.txt by default? 

If not, I can create a PR with a trivial change to the regex to pick up other files, eg just adding `(|-.+)` after `requirements` in the regex.

Just wondering if it was already a decision that was made, or if anyone has thoughts about the regex pattern to match.

(tangentially related to https://github.com/renovatebot/renovate/discussions/18171)

**Comment by rarkins** [comment]
PR welcome

**Reply by HonkingGoose** [reply]
If you're going to create a PR, please also update [`docs/usage/python.md`](https://github.com/renovatebot/renovate/edit/main/docs/usage/python.md). The document has a section that explains the default file matching regex:

> ## Alternative file names
> 
> For the `pip_requirements` manager, the default file matching regex for `requirements.txt` follows common file name conventions.
> But Renovate may not find all your files.
> 
> You can tell Renovate where to find your file(s) by setting your own `fileMatch` regex:
> 
> ```json
> {
>   "pip_requirements": {
>     "fileMatch": ["my/specifically-named.file", "\\.requirements$"]
>   }
> }
> ```

The [`python` config option docs](https://docs.renovatebot.com/configuration-options/#python) should also be updated.

And search for the word `requirements` to see what else may need to be changed. 😉
